eslint-rules/src/rules/do-not-use-theme-identifier.ts

```diff
@@ -0,0 +1,34 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+
+import { normalizePath } from '../helpers';
+
+export const doNotUseThemeIdentifierRule: TSESLint.RuleModule<string, [string]> = {
```

Comment:
Please add some documentation to explain the rule, what it does and how to configure it.